include/rocksdb/c.h

```diff
@@ -872,9 +872,9 @@ extern ROCKSDB_LIBRARY_API void rocksdb_block_based_options_set_data_block_index
     rocksdb_block_based_table_options_t*, int);  // uses one of the above enums
 extern ROCKSDB_LIBRARY_API void rocksdb_block_based_options_set_data_block_hash_ratio(
     rocksdb_block_based_table_options_t* options, double v);
-extern ROCKSDB_LIBRARY_API void
-rocksdb_block_based_options_set_hash_index_allow_collision(
-    rocksdb_block_based_table_options_t*, unsigned char);
+// rocksdb_block_based_options_set_hash_index_allow_collision()
+// is removed since BlockBasedTableOptions.hash_index_allow_collision()
+// is removed
```

Comment:
I didn't see such a notice in other removal PRs but am fine with having it. Although I didn't strongly feel it's needed so don't plan to push others to add such notice (if you do, you may wish to check all the other removal PRs that have been approved/landed lately).

Reply:
If the current public API is internally consistent, there should be no reason to talk in API docs about what used to be. History is for HISTORY.md, git log, etc.